performance-timeline/po-observe.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>PerformanceObservers: PerformanceObserverInit.buffered</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="performanceobservers.js"></script>
```

Comment:
do you actually use anything from that script? If not, I suggest removing the reference.